leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/LeshanServerDemo.java

```diff
@@ -190,8 +190,10 @@ public CaliforniumServerEndpointFactory createDefaultEndpointFactory(URI uri) {
                 return new CoapsServerEndpointFactory(uri) {
 
                     @Override
-                    protected Builder createDtlsConnectorConfigBuilder(Configuration endpointConfiguration) {
-                        Builder dtlsConfigBuilder = super.createDtlsConnectorConfigBuilder(endpointConfiguration);
+                    protected DtlsConnectorConfig.Builder createDtlsConnectorConfigBuilder(
+                            Configuration endpointConfiguration) {
+                        DtlsConnectorConfig.Builder dtlsConfigBuilder = super.createDtlsConnectorConfigBuilder(
+                                endpointConfiguration);
```

Comment:
I guess this part of code should not be changed ?

Reply:
Perhaps it was corrected by my IDE. Reverted it back.